index.html

```diff
@@ -86,25 +86,311 @@ <h3>Use-case #4: Avoiding "Hall of Mirrors"</h3>
+                      <a data-cite="SCREEN-CAPTURE#dfn-browser">browser</a>
+                      <a data-cite="SCREEN-CAPTURE#dfn-display-surface">display surface</a>'s
+                      [=top-level browsing context=]'s [=active document=] to run the
+                      following steps:
+                    </p>
+                    <ol>
+                      <li>
+                        Let |target| be the the [=relevant settings object=]'s
+                        [=environment settings object/responsible document=]'s
+                        associated navigator's {{MediaDevices}} object.
+                      <li>
+                        If |action| is not in |target|'s
+                        {{MediaDevices/[[RegisteredCaptureActions]]}}, abort these steps.
+                      <li>
+                        [=Fire an event=] named `"captureaction"`, using a
+                        {{CaptureActionEvent}} with {{CaptureActionEventInit/action}}
+                        set to |action|, at |target|.
+                      </li>
+                    </ol>
+                    <li>
+                      Wait for the event to have been fired.
+                    </li>
+                    <li>
+                      Resolve |p|.
+                    </li>
+                  </li>
+                </ol>
+              </li>
+              <li>
+                Return |p|.
+              </li>
+            </ol>
+          </dd>
+        </dl>
+        <p>
+          When a video {{MediaStreamTrack}} is created as part of the
+          <a data-cite="SCREEN-CAPTURE#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>
+          algorithm, whose source is a
+          <a data-cite="SCREEN-CAPTURE#dfn-browser">browser</a>
+          <a data-cite="SCREEN-CAPTURE#dfn-display-surface">display surface</a>,
+          give it an
+          <dfn data-dfn-for="MediaDevices">[[\AvailableCaptureActions]]</dfn> internal
+          slot, initialized to the captured
+          <a data-cite="SCREEN-CAPTURE#dfn-browser">browser</a>
+          <a data-cite="SCREEN-CAPTURE#dfn-display-surface">display surface</a>'s
+          [=top-level browsing context=]'s [=Browsing context/active window=]'s
+          associated navigator's {{MediaDevices}} object's
+          {{MediaDevices/[[RegisteredCaptureActions]]}}.
+        </p>
+        <p>
+          While capture of a
+          <a data-cite="SCREEN-CAPTURE#dfn-browser">browser</a>
+          <a data-cite="SCREEN-CAPTURE#dfn-display-surface">display surface</a>
+          is occurring, whenever that surface's
+          [=top-level browsing context=] is navigated, then for each capturer of
+          that surface, queue a task on that capturer's task-list to set all
+          associated video {{MediaStreamTrack}}s'
+          {{MediaDevices/[[AvailableCaptureActions]]}} to `[]`.
+        </p>
+      </section>
+    </section>
+    <section id="capture-handle-identification-mechanism">
```

Comment:
If Actions and Identity are in the same document, then this introduction to Identity should be encapsulated in an Identity-specific section. (Which I find very awkward, btw.)

Reply:
I'll move Identity to the top, which I hope will satisfy this. LMK if I misunderstood the concern.